charts/argocd-apps/templates/item-templates.yaml

```diff
@@ -1,4 +1,4 @@
-{{- range .Values.itemTemplates }}
+{{- range $k, _:= .Values.itemTemplates }}
```

Comment:
are you using `$k` anywhere in the file now?

Reply:
Done